app/forms/providers/proceeding_merits_task/attempts_to_settle_form.rb

```diff
@@ -0,0 +1,21 @@
+module Providers
+  module ProceedingMeritsTask
+    class AttemptsToSettleForm
+      include BaseForm
+
+      form_for ::ProceedingMeritsTask::AttemptsToSettle
+
+      attr_accessor :attempts_made, :application_proceeding_type_id
```

Comment:
just wondering why you have :application_proceeding_type_id as an attr_accessor?